src/Castle.Core.Tests/ComInterfacesTests.cs

```diff
@@ -15,14 +15,24 @@
 #if FEATURE_TEST_COM && !__MonoCS__ // Avoid loading adodb.dll
 namespace Castle.DynamicProxy.Tests
 {
+	using System;
 	using System.Reflection;
+	using System.Runtime.InteropServices;
+
 	using ADODB;
+
 	using NUnit.Framework;
 
 	[TestFixture]
 	public class ComInterfacesTests:BasePEVerifyTestCase
 	{
 		[Test]
+		public void Marshal_Release_throws_when_called_with_IntPtr_Zero()
+		{
+			Assert.Throws<ArgumentNullException>(() => Marshal.Release(IntPtr.Zero));
+		}
```

Comment:
This test probably shouldn't be in that fixture... but it felt like overkill to create a separate one for just this basic bit of .documentation.